spec.bs

```diff
@@ -2280,6 +2268,27 @@ To mend the intended distinction between [=top-level traversables=] and [=fenced
 Note: With these new definitions, a [=top-level traversable=] is essentially "unfenced" as described
 in the [[#nested-traversables-intro]].
 
+The algorithm as it exists today is a "fenced" algorithm, which will be re-introduced as a "fenced"
+variant as follows:
+
+<div algorithm>
+  A <dfn>fenced top-level traversable</dfn> is a [=traversable navigable=] with a null
+  [=navigable/parent=].
+</div>
+
+<div algorithm>
+  To get the <dfn noexport for="navigable">fenced top-level traversable</dfn> of a [=navigable=]
```

Comment:
Is `noexport` necessary here? It sounds like dfns are not exported by default: https://speced.github.io/bikeshed/#dfn-export